contrib/build-push-ceph-container-imgs.sh

```diff
@@ -64,17 +64,18 @@ function build_ceph_imgs {
 function push_ceph_imgs {
   echo "Push Ceph container image(s) to the Docker Hub registry"
   make RELEASE="$RELEASE" push.parallel
+}
 
+function build_and_push_latest_bis {
+  # latest-bis is needed by ceph-ansible so it can test the restart handlers on an image ID change
+  # rebuild latest again to get a different image ID
+  make RELEASE=master-bis FLAVORS="${CEPH_RELEASES[-1]}",centos,7 build
```

Comment:
It would be better if you could reuse BRANCH variable here to avoid hardcoded values.